### PR TITLE
Including GeneOCR UI

### PR DESCRIPTION
**Pull request by paulgeser**
Add GeneOCR UI

### CODE REVIEW

File: geneocr-ui/1.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+LABEL version="1"
+LABEL software="GeneOCR UI"
+LABEL software.version="1.0.0"
+LABEL about.summary="UI of geneocr tool, which reads gene names in images"
+LABEL about.home="https://github.com/bedapub/geneocr-ui"
+LABEL maintainer="Jitao David Zhang"
+LABEL maintainer.email="jitao_david.zhang@roche.com"
+
+# install dependencies
+RUN export DEBIAN_FRONTEND=noninteractive && \
```
Comment (osallou) [inline geneocr-ui/1.0/Dockerfile:5]: apt-get does not work on alpine

Comment (paulgeser) [reply geneocr-ui/1.0/Dockerfile:5]: I took your advise and tested it with docker locally, I will push a working version now

File: geneocr-ui/1.0/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# base image
+FROM node:13.12.0-alpine as build
+
+# metadata
+LABEL base_image="node:13.12.0-alpine"
```
Comment (osallou) [inline geneocr-ui/1.0/Dockerfile]: LABEL elts must be set in the "final" image, not the build image
In your case, after FROM nginxinc/nginx-unprivileged:stable-alpine line 43
else final image won't be tagged with labels

Comment (paulgeser) [reply geneocr-ui/1.0/Dockerfile]: Thanks for the feedback, will make the necessary changes

**Comment by osallou** [comment]
spdx license is GPL-3.0-or-later

**Comment by osallou** [comment]
@paulgeser merged, thanks for your work

**Comment by paulgeser** [comment]
Thanks for all the assistance @osallou

**Pull request merged**